apps/frontend/src/app/alert-page/component/alert-page/alert-page.component.ts

```diff
@@ -0,0 +1,164 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { SeverityType } from '../../../shared/enums/severityType';
+import {
+  Alert,
+  AlertWithBackup,
+  StorageFillAlert,
+} from '../../../shared/types/alert';
+import {
+  BehaviorSubject,
+  map,
+  Observable,
+  shareReplay,
+  Subject,
+  takeUntil,
+} from 'rxjs';
+import { ClrDatagridSortOrder, ClrDatagridStateInterface } from '@clr/angular';
+import { AlertServiceService } from '../../../shared/services/alert-service/alert-service.service';
+import { AlertUtilsService } from '../../../shared/utils/alertUtils';
+import { shortenBytes } from '../../../shared/utils/shortenBytes';
+
+interface AlertSummary {
+  criticalCount: number;
+  warningCount: number;
+  infoCount: number;
+  totalCount: number;
+  repeatedAlerts: { type: string; count: number }[];
+  mostFrequentAlert?: { type: string; count: number };
+}
+
+@Component({
+  selector: 'app-alert-page',
+  templateUrl: './alert-page.component.html',
+  styleUrl: './alert-page.component.css',
+})
+export class AlertPageComponent implements OnInit, OnDestroy {
+  protected readonly SeverityType = SeverityType;
+  readonly DAYS = 30;
+  readonly PAGE_SIZES = [10, 20, 50, 100];
+  loading = false;
+  selected: Alert[] = [];
+  hasBackupInfo = false;
+  error: string | null = null;
+  shortenBytes = shortenBytes;
+
+  criticalAlertsCount = 0;
+  warningAlertsCount = 0;
+  infoAlertsCount = 0;
+
+  private readonly alertsSubject = new BehaviorSubject<Alert[]>([]);
+  readonly alerts$ = this.alertsSubject.asObservable().pipe(shareReplay(1));
+
+  readonly alertSummary$: Observable<AlertSummary> = this.alerts$.pipe(
+    map(this.calculateAlertSummary),
+    shareReplay(1)
+  );
+
+  private readonly destroy$ = new Subject<void>();
+
+  constructor(
+    private readonly alertService: AlertServiceService,
+    private readonly alertUtils: AlertUtilsService
+  ) {}
+
+  ngOnInit(): void {
+    this.loadAlerts();
+
+    this.alertService
+      .getRefreshObservable()
+      .pipe(takeUntil(this.destroy$))
+      .subscribe(() => {
+        this.loadAlerts();
+      });
+  }
+
+  private loadAlerts(): void {
+    this.loading = true;
+    this.error = null;
+
+    this.alertService
+      .getAllAlerts(this.DAYS)
```

Comment:
```suggestion
      .getAllAlerts()
```
It should load all alerts